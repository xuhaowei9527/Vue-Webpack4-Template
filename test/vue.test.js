// 导入 Vue.js 和组件，进行测试
import { mount } from "@vue/test-utils";
import Component from "../src/test/vue.demo.vue";

describe("Component", () => {
  test("is a Vue instance", () => {
    const wrapper = mount(Component);
    expect(wrapper.element).toMatchSnapshot();
    expect(wrapper.isVueInstance()).toBeTruthy();
  });
});
