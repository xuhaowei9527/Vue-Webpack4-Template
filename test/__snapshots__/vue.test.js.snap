// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component is a Vue instance 1`] = `
<span>
  bye!
</span>
`;
